apps/dashboard/src/components/workflow-editor/configure-workflow-form.tsx

```diff
@@ -96,11 +91,11 @@ export const ConfigureWorkflowForm = (props: ConfigureWorkflowFormProps) => {
           </div>
         </SidebarHeader>
         <Separator />
-        <Form {...statusForm}>
-          <form>
+        <Form {...form}>
+          <form className="h-full" onBlur={onBlur}>
```

Comment:
Does this onBlur work? The blur [event doesn't bubble](https://developer.mozilla.org/en-US/docs/Web/API/Element/blur_event).

Reply:
It's React event bubbling not DOM
![Screenshot 2024-11-29 at 11 51 54](https://github.com/user-attachments/assets/9909cdfe-8b3c-4857-b5e5-f813e94c2190)

Reply:
Oh yes! you are right!